### DISCUSSION TITLE
Orientation Bias Filters

### DISCUSSION THREAD

**Discussion by maotian06** [uclahs-cds/pipeline-call-sSNV#65] [Ideas]
Hi all,

Here, we are trying to add a new filter function to filter out the [orientation bias](https://support.illumina.com/content/dam/illumina-support/help/Illumina_DRAGEN_Bio_IT_Platform_v3_7_1000000141465/Content/SW/Informatics/Dragen/GPipelineOrientationBiasFilter_fDG_dtSW.htm) using GATK's [LearnReadOrientationModel.](https://gatk.broadinstitute.org/hc/en-us/articles/4405451237147-LearnReadOrientationModel)

The read orientation bias is the bias that occurs during library preparation or sample processing. The most common biases are:  
1. library preparation: G -> T
2. FFPE samples: C -> T. 

When compared with the pipeline without the `LearnReadOrientationModel` function, I found little difference in the stimulated samples A-full-P2, but ~10% difference in the real FFPE sample.

https://github.com/uclahs-cds/pipeline-call-sSNV/pull/61

**Comment by maotian06** [comment]
@tyamaguchi-ucla @yashpatel6  @stefaneng @rhughwhite @jmlivingstone @zhuchcn 
Mind to test it out? But I haven't merge it to the main, so might need to use the branch `maotian-add-LearnReadOrientationModel`

**Reply by tnyamaguchi** [reply]
Were you wondering if this filter might over filter SNVs (even TPs)?
